sharding-core/sharding-core-optimize/src/main/java/org/apache/shardingsphere/core/optimize/sharding/segment/select/item/CommonSelectItem.java

```diff
@@ -37,6 +37,8 @@
     private final String expression;
     
     private final String alias;
+
+    private final String columnLabel;
```

Comment:
I think owner and name should be in this class. getExpression(), it returns owner.name, while getColumnLabel, it returns getAlias().or(getExpression()).